dev-docs/lit-release-process.md

```diff
@@ -0,0 +1,148 @@
+      1. Changelog updates to `lit` core libraries such as `lit-html` should be relected in the umbrella `lit` package.
+      1. Internal private or not ready to be released packages should not get changelogs or version bumps.
+   1. Fix errors by closing the automated "Version Packages" pull request. Commit changes to `main` branch and re-run the release process.
+
+5. Once "Version packages" is approved, (provided no new changesets were merged to `main` in the meantime), squash and merge.
+
+<blockquote>
+
+⚠️ If a commit to `main` occurred while the "Version Packages" PR was open and a new changeset was created then do not squash and merge.
+Instead re-run version packages to update the PR with the most recent changes. Otherwise the following steps will not publish a release and will instead create another "Version Packages" PR integrating the new changeset file.
+
+A publish only occurs when there are no unreleased changeset files on `main`.
+
+</blockquote>
+
+6. The process to publish to NPM and create a GitHub release follows the same process as before. Find the "Version Packages" commit and wait for at least `tests-local` to pass. Then repeat steps 1. and 2. from above, where this time the result of approving the "Release / Release" GitHub Action is publishing a new NPM version release and GitHub release.
+
+![GitHub commits image showing pending GitHub actions on Version Packages commit](./images/lit-release-process/version-packages-commit-actions.png)
+
+![GitHub Release / Release action page to deploy the Version Packages commit](./images/lit-release-process/version-packages-release-action-ui.png)
+
+![Github publish Lit release modal with "Approve and deploy" button](./images/lit-release-process/publish-lit-release-modal.png)
+
+7. Check https://github.com/lit/lit for the new release notes. Congratulations you've released Lit.
+
+# Processes to follow after a release
+
+## Updating Lit.dev generated docs
+
+If the release contains new API documentation then [lit.dev](https://lit.dev) will need to be updated.
+
+1. Copy the release commit sha.
+2. Navigate to [lit-dev-tools-cjs/src/api-docs/configs/lit-2.ts](https://github.com/lit/lit.dev/blob/7da810ec1e0a77ddea516cad3a5c9f166ce718c1/packages/lit-dev-tools-cjs/src/api-docs/configs/lit-2.ts#L24)
+3. Replace the previous sha with the new sha.
+4. Re-build Lit.dev. When the build succeeds `pages.json` and/or `symbols.json` will contain the new documentation. If these files have not been updated then no API docs have changed.
+5. Commit these changes to the lit.dev repo. [Example pull request](https://github.com/lit/lit.dev/pull/692).
+
+## Socials communication
+
+- Write a short release tweet and get it reviewed by team. Share WIP in Discord docs channel. Post to twitter linking to the changelog.
+- Link to the twitter release and post in Discord general channel.
+
+Note: If the changes are short then include a changelog image.
```

Comment:
I wonder if this socials communication process specifically needs to be documented. We've been more ad-hoc with these things. I don't think workshopping release tweet really needs to be public on discord either.

Reply:
I think it's probably worth mentioning that releases should be announced but agree that this section is a bit too full of irrelevant details. Fixed